client/thin-replica-client/src/trace_contexts.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 VMware, all rights reserved
+
```

Comment:
Does this file not need one of those license statements that begins "This product is licensed to you under..."?